### ISSUE TITLE
octant Delete Plugin

### ISSUE THREAD

**Issue by danielhelfand** [vmware-tanzu/octant#517]
Currently, you must run `rm ~/.config/octant/plugins/<plugin name>` to remove a plugin. It would be nice to be able to remove a plugin via the CLI instead so users don't have to remember the location of where plugin binaries are located.

The idea would be something along the lines of `octant delete plugin <plugin name>` would remove the plugin binary. Then that plugin will not be available when restarting Octant. 

/kind feature

**Comment by wwitzel3** [comment]
Having some mechanism for getting at installed plugins is a good idea. I think having CLI commands could work, maybe in combination with having a button in Octant that opens the plugin folder using the system file browser.

**Comment by GuessWhoSamFoo** [comment]
Plugin management as a whole should be added to the roadmap as it is something that should exist in 1.0

**Comment by danielhelfand** [comment]
@GuessWhoSamFoo Agreed. Something to also install plugins I think would be nice long term.

I think the CLI makes the most sense as the way for users to manage plugins since plugin changes require a restart anyway. Doing it through the UI might be misleading as to why something isn't added or removed.

**Comment by wwitzel3** [comment]
We would like to remove having to restart to load/unload plugins as well #178 , which ties in to what @GuessWhoSamFoo is saying about having a good plugin story as part of our 1.0 roadmap.
